.github/DISCUSSION_TEMPLATE/b-minecraft-to-discord-chat-forwarding.yml

```diff
@@ -0,0 +1,19 @@
+title: "[MC -> DC] "
+labels: ["waiting for support"]
+body:
+- type: markdown
+  attributes:
+    value: |
+      __Please create a debug report__
+
+      1. Run `/discordsrv debugger start minecraft_to_discord` to enable the debugger
+      2. Send messages in minecraft
+      3. Run `/discordsrv debugger upload` command in-game or through console
+      4. Send the link that you're given into this chat.
```

Comment:
“Into this chat” (and occurrences further down) should probably be “into this space” or “into this form” or something